tjdashboard/code/site/models/dashboards.php

```diff
@@ -0,0 +1,136 @@
+<?php
+/**
+ * @version    CVS: 1.0.0
+ * @package    Com_Tjdashboard
+ * @author     Techjoomla <extensions@techjoomla.com>
+ * @copyright  2017 Techjoomla
+ * @license    GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+// No direct access.
+defined('_JEXEC') or die;
+
+/**
+ * Methods supporting a list of TjDashboards.
+ *
+ * @since  1.0.0
+ */
+class TjdashboardModelDashboards extends JModelList
+{
+	/**
+	 * Constructor.
+	 *
+	 * @param   array  $config  An optional associative array of configuration settings.
+	 *
+	 * @see     JModelList
+	 * @since   1.0
+	 */
+	public function __construct($config = array())
+	{
+		if (empty($config['filter_fields']))
+		{
+			$config['filter_fields'] = array(
+				'dashboard_id',
+				'state',
+				'created_by',
+				'context',
+				'core',
+				'parent'
+			);
+		}
+
+		// Set the filters as default filters
+		parent::__construct($config);
+	}
+
+	/**
+	 * Get the master query for retrieving a list of requests to the senior.
+	 *
+	 * @return  JDatabaseQuery
+	 *
+	 * @since   1.0
+	 */
+	protected function getListQuery()
+	{
+		$app = JFactory::getApplication();
+
+		// Initialize variables.
+		$db    = JFactory::getDbo();
```

Comment:
- You should use the database object that is already provided by the model
- Also support a search filter to search by dashboards by name

Reply:
Done ($this)